hostinfo/hostinfo.go

```diff
@@ -153,12 +154,16 @@ func GetEnvType() EnvType {
 }
 
 var (
+	deviceTokenAtomic atomic.Value // of string (base64)
```

Comment:
what's this "base64" doing here? it's just a string, no?

base64 is a detail for another layer?